lib/brewdler/utils.rb

```diff
@@ -0,0 +1,10 @@
+module Brewdler
+  def self.system cmd, *args
+    pid = fork do
+      $stdout.reopen("/dev/null")
+      exec(cmd, *args)
+    end
```

Comment:
This may be useful (you can ignore the timeout stuff), but again, feel free to ignore it entirely: 
https://github.com/mikemcquaid/HookHand/blob/master/hookhand.rb#L123-L150